alvr/client/android/app/src/main/cpp/ovr_context.cpp

```diff
@@ -246,6 +246,15 @@ uint64_t mapButtons(ovrInputTrackedRemoteCapabilities *remoteCapabilities,
         if (remoteInputState->Touches & ovrTouch_Joystick) {
             buttons |= ALVR_BUTTON_FLAG(ALVR_INPUT_JOYSTICK_TOUCH);
         }
+        if (remoteInputState->Touches & 0x00001000) { // ovrTouch_ThumbRest
+            buttons |= ALVR_BUTTON_FLAG(ALVR_INPUT_THUMB_REST_TOUCH);
+        }
+        if (remoteInputState->Touches & 0x00002000) { // ovrTouch_LThumbRest
+            buttons |= ALVR_BUTTON_FLAG(ALVR_INPUT_THUMB_REST_TOUCH);
+        }
+        if (remoteInputState->Touches & 0x00004000) { // ovrTouch_RThumbRest
+            buttons |= ALVR_BUTTON_FLAG(ALVR_INPUT_THUMB_REST_TOUCH);
+        }
```

Comment:
Add this to the existing header?
https://github.com/alvr-org/ALVR/blob/358921292d9e7cd2830c39cd6160b885db12e840/alvr/client/android/app/include/VrApi_Input.h#L64-L82